src/experimental/test/property/arbitraries/canister_methods/pre_upgrade_method_arb.ts

```diff
@@ -6,95 +6,52 @@ import { Test } from '../../test';
 import { VoidArb } from '../candid/primitive/void_arb';
 import { Context } from '../types';
 import { UniqueIdentifierArb } from '../unique_identifier_arb';
-import {
-    BodyGenerator,
-    CanisterMethodConstraints,
-    generateMethodImplementation,
-    MethodImplementationLocationArb,
-    TestsGenerator
-} from '.';
+import { BodyGenerator, generateMethodImplementation, TestsGenerator } from '.';
 
 export type PreUpgradeMethod = {
     imports: Set<string>;
-    globalDeclarations: string[];
     sourceCode: string;
```

Comment:
Let's discuss `globalDeclarations`

Reply:
The functional syntax allowed you to globally declare your functions and then say preUpgrade(myPreUpgrade) or inspectMessage(myInspectMessage) or query(myQuery). The class api doesn't not allow for that and without that preUpgrade and inspectMessage have no use for global declarations